content/StreamingVideo.md

```diff
@@ -78,7 +78,7 @@ H="${3:-1080}"
 ./v4l2-init.sh
 echo "Streaming to ${1} with resolution ${W}x${H}..."
 
-gst-launch-1.0 v4l2src device=/dev/video0 \
+gst-launch-1.0 v4l2src device=/dev/video0 ! video/x-raw, width=1920, height=1080 \
```

Comment:
How did you choose the resolution 1920x1080?

Reply:
It is the default resolution that is used in `./v4l2-init.sh`

Reply:
ok, that works. Thanks